db/migrate/20170203141631_add_maintenance_to_storages.rb

```diff
@@ -0,0 +1,5 @@
+class AddMaintenanceToStorages < ActiveRecord::Migration[5.0]
+  def change
+    add_column :storages, :maintenance, :boolean
```

Comment:
@agrare Is this like the `read_only` and `ems_ref` values that are stored on the `host_storages` table?  If the same storage is connected to multiple providers can one provider have it in maintenance mode and one not?